### DISCUSSION TITLE
How to use avatar editor

### DISCUSSION THREAD

**Discussion by Gwendolabendooo** [dapilab/react-nice-avatar#29] [Q&A] [answered]
Hi im trying your package but i don't know how to use the avatar editor component, can you tell me how can i do it? :)

**Comment by wwayne** [comment, marked as answer]
Hi, the editor is just a demo, just trying to show the people how the avatar component can be used.
If you need to build an editor, maybe the code in here can be the reference https://github.com/dapilab/react-nice-avatar/tree/main/demo/src
